src/spectral/rulesets/.defaultsForSpectral.json

```diff
@@ -0,0 +1,27 @@
+{
```

Comment:
If we want to start with a smaller ruleset, or no rules at all, we can also do that to minimize the blast radius of this change. Either option would still give users the ability to write their own rules.